src/reducers/data.js

```diff
@@ -56,7 +56,7 @@ const stats = (state = {}, action) => {
   }
 }
 
-const byHint = (state = {}, action) => {
+const byCategory = (state = {}, action) => {
```

Comment:
https://github.com/notwaldorf/flash-cards/blob/bddf6cfa68893fe265c5e0cf177f9ca10d01f9a6/src/reducers/data.js#L52 is still referencing `byHint`

Reply:
After making this one change locally I am still not seeing mnemonics.
<img width="524" alt="screen shot 2018-05-15 at 10 59 29 am" src="https://user-images.githubusercontent.com/116360/40074689-2b2fae12-582f-11e8-8ea5-cd961fad7e03.png">

Reply:
oh hold on i think my rebase went poorly

Reply:
ok np